packages/admin-ui/ui/src/domain/orders/details/mark-shipped/index.tsx

```diff
@@ -75,7 +73,9 @@ const MarkShippedModal: React.FC<MarkShippedModalProps> = ({
       fulfillment.claim_order_id || fulfillment.swap_id || fulfillment.order_id
     const [type] = resourceId.split("_")
 
-    const tracking_numbers = data.tracking_numbers.map((tn) => tn.value)
+    const tracking_numbers = data.tracking_numbers
+      .map((tn) => tn.value)
```

Comment:
```suggestion
      .map((tn) => tn?.value)
```
not sure why the cause is, but if this is returning `undefined` as an array, we don't break

Reply:
Good catch, I don't know why this happening either in theory tn should never be undefined. But this works without having to spend to much time refactoring